addons/point_of_sale/models/pos_config.py

```diff
@@ -167,6 +167,11 @@ def _get_group_pos_user(self):
     trusted_config_ids = fields.Many2many("pos.config", relation="pos_config_trust_relation", column1="is_trusting",
                                           column2="is_trusted", string="Trusted Point of Sale Configurations",
                                           domain="[('id', '!=', pos_config_id), ('module_pos_restaurant', '=', False)]")
+    pos_is_closing_entry_with_product = fields.Boolean(
```

Comment:
```suggestion
    is_closing_entry_with_product = fields.Boolean(
```
No need to prefix the real pos.config field. Only prefix the one that mirrors it in the res.config.settings. This is because pos.config is updated thru the res.config.settings form.

Reply:
Done.